panels/site_sections/panel_applications.py

```diff
@@ -32,34 +55,72 @@ def index(self, session, message='', **params):
         panelist = session.panel_applicant(panelist_params, checkgroups=PanelApplicant.all_checkgroups, restricted=True, ignore_csrf=True)
         panelist.application = app
         panelist.submitter = True
-        other_panelists = []
-        for i in range(1, int(params.get('other_panelists', 0)) + 1):
-            applicant = {attr: params.get('{}_{}'.format(attr, i)) for attr in OTHER_PANELISTS_FIELDS}
-            other_panelists.append(PanelApplicant(application=app, **applicant))
```

Comment:
_I'm not sure where to leave this comment, so I'm just putting it here..._

I'm not sure why the creation of `other_panelists` was moved to the end of this function. It seems like you end up calling `other_panelists_from_params()` twice. Once inside `process_panel_app()` and again inside the `return` statement. It seems like it's easier to just create them once right here, no?